uspv/init.go

```diff
@@ -104,20 +103,14 @@ func (s *SPVCon) DialNode(listOfNodes []string) (net.Conn, error) {
 		} else {
 			con, err = net.Dial(conMode, conString)
 		}
-
-		if err != nil {
-			if i != len(listOfNodes)-1 {
-				log.Println(err.Error())
-				continue
-			} else if i == len(listOfNodes)-1 {
```

Comment:
we do need to catch this error or else we'll loop infinitely in case we can't connect :)

Reply:
needs change before merge :)